standard/info/wikis/freefire/info.lua

```diff
@@ -29,4 +29,5 @@ return {
 	defaultRoundPrecision = 0,
 	defaultTeamLogo = 'Freefire Default Logo.png', ---@deprecated
 	defaultTeamLogoDark = 'Freefire Default Logo.png', ---@deprecated
+	match2 = 0,
```

Comment:
has some setup for it and usage on 8 pages
but not on git

Reply:
as above